adaptive_navigation/example/lib/main.dart

```diff
@@ -1,5 +1,7 @@
+import 'package:adaptive_navigation_example/custom_scaffold.dart';
 import 'package:flutter/material.dart';
-import 'package:adaptive_navigation/scaffold.dart';
+
+import 'default_scaffold.dart';
```

Comment:
Either this should be an absolute import or the other should be a relative for consistency.